### PR TITLE
WIP Require external lib where used

### PR DESCRIPTION
**Pull request by ger-benjamin**
For https://jira.camptocamp.com/browse/GSGMF-200

### CODE REVIEW

File: contribs/gmf/src/layertree/timesliderComponent.js [hunk 1]
```diff
@@ -2,13 +2,14 @@ goog.provide('gmf.layertree.timeSliderComponent');
 
 goog.require('gmf');
 goog.require('ngeo.misc.WMSTime');
-
+// webpack: import 'angular-ui-slider/src/slider.js';
```
Comment (sbrunner) [inline contribs/gmf/src/layertree/timesliderComponent.js]: Can't just be:
```
// webpack: import 'angular-ui-slider';
```

File: contribs/gmf/src/query/windowComponent.js [hunk 2]
```diff
@@ -13,6 +13,9 @@ goog.require('ol.style.Circle');
 goog.require('ol.style.Fill');
 goog.require('ol.style.Stroke');
 goog.require('ol.style.Style');
+// webpack: import 'angular-animate/angular-animate.js';
+// webpack: import 'angular-touch/angular-touch.js';
```
Comment (sbrunner) [inline contribs/gmf/src/query/windowComponent.js]: => ? 
```
// webpack: import 'angular-animate';
// webpack: import 'angular-touch';
```

File: examples/measure.js [hunk 3]
```diff
@@ -21,12 +21,15 @@ goog.require('ol.style.Style');
 goog.require('ol.style.Circle');
 goog.require('ol.style.Stroke');
 goog.require('ol.style.Fill');
+// webpack: import 'angular-sanitize/angular-sanitize.js';
```
Comment (sbrunner) [inline examples/measure.js]: same

File: src/download/service.js [hunk 4]
```diff
@@ -2,6 +2,8 @@ goog.provide('ngeo.download.service');
 
 goog.require('ngeo');
 goog.require('ngeo.utils');
+// webpack: import * as FileSaver from 'file-saver/FileSaver.js';
+// webpack: const saveAs = FileSaver.saveAs;
```
Comment (sbrunner) [inline src/download/service.js]: => ?
```
// webpack: import {saveAs} from 'file-saver';
```

File: src/message/displaywindowComponent.js [hunk 5]
```diff
@@ -1,14 +1,19 @@
 goog.provide('ngeo.message.displaywindowComponent');
 
 goog.require('ngeo');
+// webpack: import 'angular-sanitize/angular-sanitize.js';
```
Comment (sbrunner) [inline src/message/displaywindowComponent.js]: same

File: src/message/popupcomponent.js [hunk 6]
```diff
@@ -1,10 +1,15 @@
 goog.provide('ngeo.message.popupComponent');
 
 goog.require('ngeo');
+// webpack: import 'angular-sanitize/angular-sanitize.js';
```
Comment (sbrunner) [inline src/message/popupcomponent.js]: same

**Comment by ger-benjamin** [comment]
So, tests passes but I can't test. I'm not sure of all my code (nothing hard to debug if that doesn't work).

! I've not done anything for application directly. Do you think I miss something ?.

**Pull request merged**